tmdb/season.go

```diff
@@ -170,6 +171,12 @@ func (season *Season) ToListItem(show *Show) *xbmc.ListItem {
 			Genre:         show.GetGenres(),
 			Studio:        show.GetStudios(),
 		},
+		Properties: map[string]string{
+			"TotalEpisodes": strconv.Itoa(season.EpisodeCount),
+			// TODO: calculate somehow these
```

Comment:
All episodes/seasons/shows/movies from Trakt+Kodi are collected to `playcount` map, example of usage:
https://github.com/elgatito/elementum/blob/master/tmdb/episode.go#L116
Don't know how much will it take to calculate those numbers on the fly.

Reply:
done. some speed-related info is below.